combat-trainer.lic

```diff
@@ -1999,6 +2036,62 @@ class GameState
     @attack_overrides[weapon_skill]
   end
 
+  def set_currently_whirlwinding(setting)
+    @currently_whirlwinding = setting
+  end
+
+  def currently_whirlwinding?
+    @currently_whirlwinding
+  end
+
+  def whirlwind_trainables
+    @whirlwind_trainables
+  end
+
+  def whirlwind_offhand_name
+    weapon_training[@current_whirlwind_offhand_skill]
+  end
+
+  def last_whirlwind_offhand_weapon_name
```

Comment:
It's not used, but thought to input it as we have it for last_weapon, etc. Can be removed without an issue.